rclpy/rclpy/node.py

```diff
@@ -27,14 +27,16 @@ def __init__(self, handle):
     def create_publisher(self, msg_type, topic, qos_profile):
```

Comment:
fixed in https://github.com/ros2/rclpy/pull/39/commits/7d679518e3caa4413130276b222b1d76c5adf475